doc-content/enterprise-only/project-data/data-objects-create-proc.adoc

```diff
@@ -12,12 +12,12 @@ The following procedure is a generic overview of creating data objects. It is no
 .Procedure
 . In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project name.
 . Click *Add Asset* → *Data Object*.
-. Enter a unique *Data Object* name and select the *Package* where you want the data object to be available for other rule assets. Data objects with the same name cannot exist in the same package. The package that you specify must be the same package where the rule assets that require those data objects have been assigned or will be assigned.
+. Enter a unique *Data Object* name and select the *Package* where you want the data object to be available for other rule assets. Data objects with the same name cannot exist in the same package. In the DRL file you specify, you can import a data object from any package.
 +
 .Importing data objects from other packages
 [IMPORTANT]
 ====
-You can also import an existing data object from another package into the package of the rule asset. Select the relevant rule asset within the project and in the asset designer, go to *Data Objects -> New item* to select the object to be imported.
+You can also import an existing data object from another package directly into the asset designer. Select the relevant rule asset within the project and in the asset designer, go to *Data Objects -> New item* to select the object to be imported.
```

Comment:
The "also" here at the beginning can be avoided. If you really want to emphasize an additional feature, instead of using "also", we can say, "In addition to xyz, you can do abc also..".

Reply:
Changed as suggested.